stdlib/2/dircache.pyi

```diff
@@ -0,0 +1,10 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/dircache.py
+
+from typing import List
+
+def reset() -> None: ...
+def listdir(path: str) -> List[str]: ...
+
+opendir = listdir
+
+def annotate(head: str, list: List[str]) -> None: ...
```

Comment:
But `List[Text]` is problematic because it's incompatible with `List[str]` due to invariance. I think this should actually be `Union[List[str], List[Text], List[Union[str, Text]]]` in Python 2.